test/e2e/volumes.go

```diff
@@ -79,7 +79,7 @@ func startVolumeServer(client *client.Client, config VolumeTestConfig) *api.Pod
 	serverPod := &api.Pod{
 		TypeMeta: api.TypeMeta{
 			Kind:       "Pod",
-			APIVersion: "v1beta3",
+			APIVersion: "v1",
```

Comment:
Yes. I also mentioned that here: https://github.com/GoogleCloudPlatform/kubernetes/pull/9301/files#r31786519

I'm fine for that to be a subsequent PR.